README.md

```diff
@@ -103,41 +45,71 @@ graph LR
     E_1 -.-> E_0
```

Comment:
I don't understand why there is to 'update current room' if we are checking if there is space for a new room? Shouldn't this be 'create a room' ?